File: codigos/1 - estrutura sequencial/1012 - Area/Main.cs
using System; 
using static System.Console;

// class URI {
    static double PI = 3.14159;

    static string triangulo(double A, double C){
        double area = A*C/2.0;
        return Math.Round(area, 3).ToString("F3");
    }

    static string circulo(double C){
        double area = PI * Math.Pow(C, 2);
        return Math.Round(area, 3).ToString("F3");
    }

    static string trapezio(double A, double B, double C){
        double area = (A+B)*C/2.0;
        return Math.Round(area, 3).ToString("F3");
    }

    static string retangulo(double X, double Y){
        double area = X*Y;
        return Math.Round(area, 3).ToString("F3");
    }

    static void exibirSaidas(double[] valores){
        double A = valores[0];
        double B = valores[1];
        double C = valores[2];

        WriteLine($"TRIANGULO: {triangulo(A,C)}");
        WriteLine($"CIRCULO: {circulo(C)}");
        WriteLine($"TRAPEZIO: {trapezio(A,B,C)}");
        WriteLine($"QUADRADO: {retangulo(B,B)}");
        WriteLine($"RETANGULO: {retangulo(A,B)}");
    }
    // static void Main(string[] args) { 
        string[] linha = ReadLine().Split(' ');
        double[] valores = Array.ConvertAll(linha, x => double.Parse(x));
        exibirSaidas(valores);
    // }
// }

File: codigos/1 - estrutura sequencial/1012 - Area/Main.dart
import 'dart:io';
import 'dart:math';

double PI = 3.14159;

double TRIANGULO(double A, double C){
  return A*C/2;
}

double CIRCULO(double C){
  return PI * pow(C, 2);
}

double TRAPEZIO(double A, double B, double C){
  return (A+B)*C/2;
}

double RETANGULO(double X, double Y){
  return X*Y;
}

exibirSaidas(List<double> entrada){
  double A = entrada[0];
  double B = entrada[1];
  double C = entrada[2];

  String triangulo = TRIANGULO(A,C).toStringAsFixed(3);
  String circulo = CIRCULO(C).toStringAsFixed(3);
  String trapezio = TRAPEZIO(A,B,C).toStringAsFixed(3);
  String quadrado = RETANGULO(B,B).toStringAsFixed(3);
  String retangulo = RETANGULO(A,B).toStringAsFixed(3);

  stdout.writeln("TRIANGULO: $triangulo");
  stdout.writeln("CIRCULO: $circulo");
  stdout.writeln("TRAPEZIO: $trapezio");
  stdout.writeln("QUADRADO: $quadrado");
  stdout.writeln("RETANGULO: $retangulo");
}

main(){
  List<double> entrada = stdin
                      .readLineSync()!
                      .split(' ')
                      .map((x) => double.parse(x))
                      .toList();

  exibirSaidas(entrada);
}